services/inc/services_dynalib.h

```diff
@@ -96,6 +97,8 @@ DYNALIB_FN(48, services, devicetree_tree_lock, int(void*))
 DYNALIB_FN(49, services, devicetree_tree_get, int(void*, uint32_t, void*))
 DYNALIB_FN(50, services, devicetree_string_dictionary_lookup, const char*(uint32_t, void*))
 DYNALIB_FN(51, services, devicetree_hash_string, uint32_t(const char*, size_t))
+DYNALIB_FN(52, services, security_mode_set, int(module_info_security_mode, void*))
```

Comment:
Why do we need security_mode_set in dynalib? Shouldn't the bootloader be the only caller of this method?

Reply:
Needed for testing, it also doesn't allow protected->non-protected, so should be fine, but I'll remove this.